easybuild/easyconfigs/a/any2fasta/any2fasta-0.4.2-GCC-8.2.0-2.31.1-Perl-5.28.1.eb

```diff
@@ -0,0 +1,35 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'Tarball'
+
+name = 'any2fasta'
+version = '0.4.2'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'https://github.com/tseemann/any2fasta'
+description = "Convert various sequence formats to FASTA"
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
```

Comment:
Why include a GCC and GCCcore version?  Since it's a straight up perl code anyway, I can't see how there would be any performance difference, so GCCcore should suffice.

Reply:
thanks, forgot to remote old one :)